lib/internal/modules/cjs/loader.js

```diff
@@ -188,6 +190,11 @@ for (const { 0: id, 1: mod } of NativeModule.map) {
   }
 }
 
+const allBuiltins = new SafeSet(
```

Comment:
can we just export this set instead of the `isBuiltin` function?

Reply:
It would be mutable and that does not seem ideal. I would therefore not expose the set.